documentation/src/docs/index.md

```diff
@@ -15,22 +15,21 @@ Read [Pravega Concepts](pravega-concepts.md) for more details.
 
 ## Key Features 
 
--   Exactly-Once Semantics - Ensure that each event is delivered and processed exactly once, with exact ordering guarantees, despite failures in clients, servers or the network.
+-   Exactly-Once Semantics:- Pravega ensures that each event is delivered and processed exactly once, with exact ordering guarantees, despite failures in clients, servers or the network.
 
--   Auto Scaling - Unlike systems with static partitioning, Pravega can automatically scale individual data streams to accommodate changes in data ingestion rate.
+-   Auto Scaling:- Pravega performs auto scaling by automatically scaling the individual data streams to accommodate changes in data ingestion rate.
 
--   Distributed Computing Primitive - Pravega is great for distributed computing; it can be used as a data storage mechanism, for messaging between processes and for other distributed computing services such as leader election.
+-   Distributed Computing Primitive:- Pravega is very efficient in performing distributed computing. It can be used as a data storage mechanism. It can be used for messaging between processes and for other distributed computing services such as leader election.
 
--   Write Efficiency - Pravega shrinks write latency to milliseconds, and seamlessly scales to handle high throughput reads and writes from thousands of concurrent clients, making it ideal for IoT and other time sensitive applications.
+-   Write Efficiency:- Pravega shrinks write latency to milliseconds, and seamlessly scales to handle high throughput reads and writes from thousands of concurrent clients, making it ideal for IoT and other time sensitive applications.
 
--   Unlimited Retention - Ingest, process and retain data in streams forever. Use same paradigm to access both real-time and historical events stored in Pravega.
+-   Unlimited Retention:- Pravega performs ingestion, processing and retains data in streams forever. Pravega uses the same paradigm to access both real-time and historical events stored in it.
 
--   Storage Efficiency - Use Pravega to build pipelines of data processing, combining batch, real-time and other applications without duplicating data for every step of the pipeline.
+-   Storage Efficiency:-  Pravega is used to build pipelines of data processing. It is used to combine batch, real-time and other applications without duplicating data for every step of the pipeline.
 
--   Durability - Don't compromise between performance, durability and consistency.
-    Pravega persists and protects data before the write operation is acknowledged to the client.
+-   Durability:- Pravega do not compromise between performance, durability and consistency. Pravega persists and protects data before the write operation is acknowledged to the client.
```

Comment:
Typo: `Pravega does not`. Moreover, I don't like the first sentence, because it is probably not true. There is an inherent trade-off relating performance and durability/consistency. That is, if you want durability and consistency guarantees, this will probably entail a set of tasks to be done that will necessary make the system slower compared to another one that does not provide such guarantees. But not only this, the second sentence says `before the write operation is acknowledged to the client`; I think that this is not true neither. If a writer writes an events and gets disconnected before getting the acknowledgement, there is not guarantee that the server has correctly stored this data event. Conversely, once an event is confirmed to the writer, we are sure that it has been correctly persisted. 

I would change this bullet as follows:
`Durability: Pravega persists and protects data events once the write operation is acknowledged to the client.`